packages/gatsby-theme-newrelic/src/components/__tests__/SplitTextButton.test.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { SplitFactory } from '@splitsoftware/splitio-react';
+import SplitTextButton from '../SplitTextButton';
+import { renderWithProviders } from '../../test-utils/renderHelpers';
+import { screen } from '@testing-library/react';
+
+jest.mock('gatsby', () => ({
+ __esModule: true,
```

Comment:
Super nitpicky: it looks like this block is only indented in _one_ space. Per [our underlying eslint configuration](https://github.com/newrelic/eslint-plugin-newrelic/blob/b975e5d29bc28692fdecf94b763e3ceb09c2e6ee/lib/config/rules/stylistic-issues.js#L109-L111), we try to have _two_ spaces.

Reply:
Thanks. Fixed this.